### PR TITLE
Add public holidays for Ghana

### PR DESCRIPTION
**Pull request by petersowah**
# Contributing a new country ?

* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/spatie/holidays/pulls) for the same country?

1. Create a new class in the Countries directory. It should extend the Country class.
2. Add a test for the new country in the tests directory.
3. Run the tests so a snapshot gets created.
4. Verify the result in the newly created snapshot is correct.

### CODE REVIEW

File: src/Countries/Ghana.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            'Christmas Day' => '12-25',
+            'Boxing Day' => '12-26',
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+        $easter = CarbonImmutable::createFromTimestamp(easter_date($year))
+            ->setTimezone('Africa/Accra');
```
Comment (Nielsvanpach) [inline src/Countries/Ghana.php:33]:
```suggestion
        $easter = $this-easter($year);
```

File: tests/Countries/GhanaTest.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    $holidays = Holidays::for(country: 'gh')->get();
+
+    expect($holidays)
+        ->toBeArray()
+        ->not()->toBeEmpty();
+
+    expect(formatDates($holidays))->toMatchSnapshot();
+});
+
+it('can calculate Ghana date based regional holidays', function () {
```
Comment (Nielsvanpach) [inline tests/Countries/GhanaTest.php:33]: No regional holidays are added yet to this PR?

File: tests/Countries/GhanaTest.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    $holidays = Holidays::for(country: 'gh')->get();
+
+    expect($holidays)
+        ->toBeArray()
+        ->not()->toBeEmpty();
+
+    expect(formatDates($holidays))->toMatchSnapshot();
+});
+
+it('can calculate Ghana easter based region holidays', function () {
```
Comment (Nielsvanpach) [inline tests/Countries/GhanaTest.php:21]: No regional holidays are added yet to this PR?

File: tests/Countries/GhanaTest.php (hunk 2 above)
Comment (petersowah) [reply tests/Countries/GhanaTest.php:33]: This is removed from the tests now. Thanks

**Comment by Nielsvanpach** [comment]
Thanks!

**Pull request merged**